addons/ccm-azure/ccm-azure.yaml

```diff
@@ -155,23 +154,20 @@ spec:
       priorityClassName: system-node-critical
       hostNetwork: true
       serviceAccountName: cloud-controller-manager
+      nodeSelector:
+        node-role.kubernetes.io/control-plane: ""
       tolerations:
-        - key: node-role.kubernetes.io/control-plane
-          effect: NoSchedule
         - key: node-role.kubernetes.io/master
           effect: NoSchedule
-        - key: "node.cloudprovider.kubernetes.io/uninitialized"
```

Comment:
I think we need this toleration, how else the CCM would run?

Reply:
The upstream manifest doesn't have this annotation any longer and it works without it (I tested it manually), so I don't think it's needed at all.

Reply:
Super strange... I suppose some other component initializes the node (node-agent maybe?).